qemu/tests/cfg/msi_change_flag.cfg

```diff
@@ -0,0 +1,33 @@
+- msi_change_flag:
+    only virtio_net
+    type = msi_change_flag
+    #In this test need set snapshot for our test will chang guest msi support
+    check_image = yes
+    clone_master = None
+    filesize = 4000
+    file_md5_check_timeout = 600
+    dd_cmd = "dd if=/dev/zero of=%s oflag=direct bs=1M count=%d"
+    Linux:
+        tmp_dir = "/var/tmp/"
+    Windows:
+        tmp_dir = "C:\\"
+        i386:
+            devcon_folder = "WIN_UTILS:\devcon\x86\"
+        x86_64:
+            devcon_folder = "WIN_UTILS:\devcon\amd64\"
+    variants:
+        - disable_pci_msi:
+            only Linux
+            disable_pci_msi = yes
+            #after set pci_nomsi, we need kill the vm.
+            kill_vm = yes
```

Comment:
Can delete this line of comments because using `disable_pci_msi`

Reply:
ok,I will delete it.